lib/tasks/jenkins.rake

```diff
@@ -1,25 +1,40 @@
 begin
   require "ci/reporter/rake/minitest"
 
-  namespace :foreman do
-    task :set_test_runner do
-      ENV['TESTOPTS'] = "#{ENV['TESTOPTS']} #{Rails.root}/test/test_runner.rb"
+  namespace :jenkins do
+    task :unit do
+      Rails.env = 'test'
+      Rake::Task['db:migrate'].invoke
+      system "rake jenkins:setup:minitest test RAILS_ENV=test"
```

Comment:
Why can't the rake tasks be called directly?  Running an external command will create an additional Ruby environment.